proxy/src/control/destination/mod.rs

```diff
@@ -228,10 +242,40 @@ impl Responder {
 
 // ===== impl Metadata =====
 
+impl Default for Metadata {
+    fn default() -> Self {
+        Self::no_metadata()
+    }
+}
+
 impl Metadata {
-    fn no_metadata() -> Self {
+    /// Construct a Metadata struct representing an endpoint with no metadata.
+    pub fn no_metadata() -> Self {
         Metadata {
-            metric_labels: None,
+            dst_labels: None,
+            // If we have no metadata on an endpoint, assume it does not support TLS.
+            tls_identity: None,
         }
     }
+
+    /// Construct a new Metadata with a set of labels from the Destination service.
```

Comment:
I think we should just ditch all these comments. They add no value and some of them are already wrong or incomplete. For example, this one doesn't mention the TLS identity.